lib/interfaces/charts.js

```diff
@@ -155,7 +160,7 @@ function fileToMapInfo(file) {
           reject(err);
         }
         if (_.isUndefined(mbtilesData.name)) {
-          resolve(undefined);
+          reject(err);
```

Comment:
If we resolve this as undefined, we get into problems further down, as mbtilesData.name is called again